src/App/Activity/Model/TotaluseractivityModel.php

```diff
@@ -58,8 +58,6 @@ public function getTotalActivity()
 		];
 
 		$periodList  = [1 => 7, 2 => 30, 3 => 90];
-		$periodNames = [1 => 'Weeks', 2 => 'Months', 3 => 'Quarters'];
```

Comment:
Wasn't getting used at all actually.  Probably leftover copy/paste when I was pulling code out of the original CMS component.  The translations are all being done in the controllers.